crates/interpreter/CHANGELOG.md

```diff
@@ -55,4 +55,4 @@
 
 ## 0.1.0
 
-<!-- Increment to skip CHANGELOG.md test: 4 -->
+<!-- Increment to skip CHANGELOG.md test: 5 -->
```

Comment:
Sorry I missed that file. Actually this crate is published so you need to update the changelog. However, we could do it later, so let's keep it like that for now.